api/src/main/java/org/openmrs/api/impl/UserServiceImpl.java

```diff
@@ -67,6 +73,16 @@ public void setUserDAO(UserDAO dao) {
 		this.dao = dao;
 	}
 	
+	
+	/**
+	 * @return the validTime for which the password reset activation key will be valid
+	 */
+	private int getValidTime() {
+		int validTime= Integer.parseInt(Context.getAdministrationService().getGlobalProperty("validTime"));
```

Comment:
What happens if the value isn't set? i.e this returns null or blank string, won't you get a NPE or cast exception